docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Joining Nodes via Azure Managed Identity
+description: Use the Azure join method to add Nodes to your Teleport cluster on Azure
+---
+
+This guide will explain how to use the **Azure join method** to configure
+Teleport Nodes and Proxy Service instances to join your Teleport cluster without
+sharing any secrets when they are running in an Azure Virtual Machine.
+
+The Azure join method is available in Teleport 12.1+. It is available to any
+Teleport Node or Proxy running in an Azure Virtual Machine. The Virtual Machine
+must have a
+[Managed Identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
+assigned to it with permission to read virtual machine info.
+
+<Details
+scope={["oss", "enterprise", "cloud"]}
+scopeOnly
+opened
+title="Another Azure Node joining method"
+>
+
+You can also configure Nodes running in Azure to join a cluster via **secret tokens**,
+which is useful when you don't want to rely on Azure-specific APIs.
+Read more in the following guide:
+[Adding Nodes to the cluster](../admin/adding-nodes.mdx)
+
+</Details>
+
+<Notice type="warning" scope={["oss", "enterprise"]}>
+
+The Azure join method will not work if TLS is terminated at a load balancer in
+front of your Teleport Proxy Service unless the Node using this method is
+connecting directly to the Auth Service.
+
+</Notice>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An Azure Virtual Machine to act as a Teleport Node, with the Teleport binary
+  installed.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/4. Set up Managed Identity.
+Every virtual machine hosting a Node or Proxy using the Azure method to join
+your Teleport cluster needs a Managed Identity assigned to it. The identity 
+requires at least the `Microsoft.Compute/virtualMachines/read` permission.
```

Comment:
> The identity 
requires at least the...

Do we have any guidance on when a user should add a permission besides `Microsoft.Compute/virtualMachines/read` to the Managed Identity?

I think it would also make this paragraph a bit clearer to indicate how the Node/Proxy uses the `Microsoft.Compute/virtualMachines/read` permission.